### PR TITLE
Lab 2 adding quiz questions

### PR DESCRIPTION
**Pull request by aramirezdecruz3148**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>A webpage about me</title>
+</head>
+<body>
+    <header>
+        <h1>Everything You Need to Know About Alex</h1>
+
+        <img id="image" src="https://d2g8igdw686xgo.cloudfront.net/37131652_1550357560852911_r.jpeg">
```
Comment (Katedam) [inline index.html:14]: Make sure to always include a descriptive alt attribute on img tags for accessibility and for those times when images won't load like slow connections, etc.

File: src/score-result.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    qOne = qOne.toLowerCase();
+    qTwo = qTwo.toLowerCase();
+    qThree = qThree.toLowerCase();
+    qFour = qFour.toLowerCase();
+    qFive = qFive.toLowerCase();
+
+    let score = 0;
+
+    if(qOne === 'bugs') {
+        score += 1;
```
Comment (Katedam) [inline src/score-result.js:11]: Quick tip: if you are adding in increments of 1 you can just put `score++` instead of `score += 1`

Comment (aramirezdecruz3148) [reply src/score-result.js:11]: Awesome thanks!

File: index.html (hunk 1 above)
Comment (aramirezdecruz3148) [reply index.html:14]: Thank you, I totally forgot!

**Pull request merged**